README.md

```diff
@@ -1,13 +1,29 @@
-# Universal Connect Project UI
+# Universal Connect Project - Hosted Applications
 
-This repo is a monorepo using [turborepo](https://turbo.build/repo) for the Universal Connect Project app
+This is a monorepo for the UCP-hosted applications and services. It uses [TurboRepo](https://turbo.build/repo)
 
-## Getting Started (in development)
+These applications are hosted by the Universal Connect Project, and are not intended to be hosted by a customer. The [UCW-App](https://github.com/Universal-Connect-Project/ucw-app) application, which is a customer-hosted service, interacts with these UCP-hosted services. UCP-App has been built in such a way that it does rely on these hosted services in order to function as expected. It can use them to update data, but does not require them.
 
-1. clone the repo
-1. Run `npm install`
+This monorepo contains the following applications:
+
+- **[Authentication Service](./apps/authentication-service/README.md)** - This service consists of client and server authentication for the Universal Connect Project. It uses Auth0 as a backend.
```

Comment:
This makes it sound like the Auth Service is what is doing our authentication

Reply:
Wording updated.